### PR TITLE
feat: Add realtime icon to trip leg times

### PR DESCRIPTION
**Pull request by ingvisim**
Add realtime icon to trip leg times to remove "ca." and differenciate between no realtime and trip that has not passed any stops.

<details>
<summary>Screenshot</summary>

Simulator Screen Shot - iPhone 14 Pro - 2022-12-22 at 08 28 46
</details>

https://github.com/AtB-AS/kundevendt/issues/2949

### CODE REVIEW

File: src/screens/TripDetails/components/Time.tsx [hunk 1]
```diff
@@ -5,24 +5,43 @@ import {formatToClock} from '@atb/utils/date';
...
-          <AccessibleText
-            prefix={t(dictionary.travel.time.expectedPrefix)}
-            testID="expTime"
-          >
-            {expected}
-          </AccessibleText>
+          <View style={{flexDirection: 'row', alignItems: 'center'}}>
+            {showRealtime && realtimeIcon}
+            <AccessibleText
+              prefix={t(dictionary.travel.time.expectedPrefix)}
```
Comment (gorandalum) [inline src/screens/TripDetails/components/Time.tsx:39]: As far as I can see `expectedPrefix` says "Forventet tid" and not "Sanntid". Should this be changed?

File: src/screens/TripDetails/DepartureDetails/index.tsx [hunk 2]
```diff
@@ -320,9 +320,12 @@ function EstimatedCallRow({
...
           <Time
-            aimedTime={call.aimedDepartureTime}
-            expectedTime={call.expectedDepartureTime}
-            missingRealTime={!call.realtime && isStartOfServiceJourney}
+            timeValues={{
+              aimedTime: call.aimedDepartureTime,
+              expectedTime: call.expectedDepartureTime,
+              missingRealTime: !call.realtime && isStartOfServiceJourney,
+            }}
+            showRealtime={false}
```
Comment (rosvik) [inline src/screens/TripDetails/DepartureDetails/index.tsx:328]: Probably clearer to call this `showRealtimeIcon` (Sorry, my fault 🙈)

File: src/screens/TripDetails/components/Time.tsx [hunk 3]
```diff
@@ -36,15 +55,15 @@ const Time: React.FC<TimeValues> = (timeValues) => {
...
-        </ThemeText>
-      );
+      return <ThemeText testID="schCaTime">{scheduled}</ThemeText>;
     }
     default: {
-      return <ThemeText testID="schTime">{scheduled}</ThemeText>;
+      return (
+        <View style={{flexDirection: 'row', alignItems: 'center'}}>
+          {showRealtime && realtimeIcon}
+          <ThemeText testID="schTime">{scheduled}</ThemeText>
```
Comment (gorandalum) [inline src/screens/TripDetails/components/Time.tsx:64]: I know you haven't touched this, but I think we should show the `expected` time here. Even though `scheduled` and `expected` is pretty much the same in this situation, the `expected` time will be most correct, and might deviate with up to a minute from the `scheduled` time.

File: src/screens/TripDetails/components/Time.tsx [hunk 4]
```diff
@@ -5,24 +5,43 @@ import {formatToClock} from '@atb/utils/date';
...
   const representationType = getTimeRepresentationType(timeValues);
   const scheduled = formatToClock(aimedTime, language);
   const expected = expectedTime ? formatToClock(expectedTime, language) : '';
 
+  const realtimeIcon = (
+    <ThemeIcon
+      svg={themeName == 'dark' ? RealtimeDark : RealtimeLight}
+      size="small"
+      style={{marginRight: 4}}
+    ></ThemeIcon>
```
Comment (rosvik) [inline src/screens/TripDetails/components/Time.tsx:29]:
```suggestion
    />
```

**Review by gorandalum — APPROVED** [review]

File: src/screens/TripDetails/components/Time.tsx [hunk 5]
```diff
@@ -5,24 +5,43 @@ import {formatToClock} from '@atb/utils/date';
...
   const {aimedTime, expectedTime} = timeValues;
   const representationType = getTimeRepresentationType(timeValues);
   const scheduled = formatToClock(aimedTime, language);
   const expected = expectedTime ? formatToClock(expectedTime, language) : '';
 
+  const realtimeIcon = (
+    <ThemeIcon
+      svg={themeName == 'dark' ? RealtimeDark : RealtimeLight}
+      size="small"
+      style={{marginRight: 4}}
```
Comment (rosvik) [inline src/screens/TripDetails/components/Time.tsx:28]:
```suggestion
  const {themeName, theme} = useTheme();
  const {aimedTime, expectedTime} = timeValues;
  const representationType = getTimeRepresentationType(timeValues);
  const scheduled = formatToClock(aimedTime, language);
  const expected = expectedTime ? formatToClock(expectedTime, language) : '';

  const realtimeIcon = (
    <ThemeIcon
      svg={themeName == 'dark' ? RealtimeDark : RealtimeLight}
      size="small"
      style={{marginRight: theme.spacings.xSmall}}
```

**Review by rosvik — APPROVED** [review]
Looks good 💯 Noticed one more nitpick. You should be able to just use "Commit suggestion" tho.

**Review by gorandalum — APPROVED** [review]

**Pull request merged**